### ISSUE TITLE
Syntax error in program "ZCL_ABAPGIT_GIT_TRANSPORT=====CP ".

### ISSUE THREAD

**Issue by patvo** [larshp/abapGit#2462]
Hi there.
Getting a syntax error in ZCL_ABAPGIT_GIT_TRANSPORT (Line 101) only on our NW 7.31 (SP0007) machine. 
[abapGit_syntax_error_20190226.txt](https://github.com/larshp/abapGit/files/2907551/abapGit_syntax_error_20190226.txt)

**Comment by larshp** [comment]
is the version in sync? the method exists, https://github.com/larshp/abapGit/search?q=CHECK_SMART_RESPONSE&unscoped_q=CHECK_SMART_RESPONSE

```
     The following syntax error occurred in program
      "ZCL_ABAPGIT_GIT_TRANSPORT=====CP " in include
      "ZCL_ABAPGIT_GIT_TRANSPORT=====CM002 " in
     line 13:
     "Die Methode "CHECK_SMART_RESPONSE" ist unbekannt bzw. PROTECTED oder P"
     "RIVATE."
     " "
     " "
```

**Comment by patvo** [comment]
@larshp 
Hi Lars.
Seems like this is on my end.
I recently updated a NW 7.51 machine from abapGit Version 1.76.0 to 1.85.0 without any problems.
I did the same online update on a NW 7.31 machine also from 1.76.0 to 1.85.0. However, the class zcl_abapgit_http_client was in the main package and not in zabapgit_http. So I didn’t find it at first. Don’t know how this could happen. Fixed it manually, now it’s all fine.
Many thanks for the support and sorry for opening an issue.
Patrick
